homeassistant/components/matrix/__init__.py

```diff
@@ -82,9 +89,8 @@
 )
 
 
-def setup(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
The `setup` method has been removed, but we would still need an integration setup, to allow triggering an import flow to import existing YAML configuration into the UI.